### PR TITLE
Adds basic read the docs integration

### LINKED ISSUE #43: Readthedocs integration

### PR DESCRIPTION
**Pull request by fatboystring**
Closes #43

### CODE REVIEW

File: docs/getting_started/index.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    $ pip install omniforms
+
+(``sudo`` may be required if installing system-wide or without virtualenv)
+
+Once Omniforms is installed into your python environment, you will need to add the omniforms library to the  ``INSTALLED_APPS`` in your django projects settings file.
+
+.. code-block:: python
+
+    INSTALLED_APPS += ['omni_forms']
```
Comment (dammitjim) [inline docs/getting_started/index.rst]: I had to register this as `omniforms`

File: docs/wagtail/index.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+===================
+
+Installation
+------------
+
+In addition to the steps outlined in the 'Getting started' section you will need to add the ``omniforms.wagtail`` app to your projects ``INSTALLED_APPS`` setting.
+
+.. code-block:: python
+
+    INSTALLED_APPS += ['omni_forms', 'omni_forms.wagtail']
```
Comment (dammitjim) [inline docs/wagtail/index.rst]: I had to register this as omniforms

File: docs/getting_started/index.rst (hunk 1 above)
Comment (fatboystring) [reply docs/getting_started/index.rst]: fixed

File: docs/wagtail/index.rst (hunk 2 above)
Comment (fatboystring) [reply docs/wagtail/index.rst]: fixed

**Comment by dammitjim** [comment]
It would be cool to have some docs on the built-in handlers available.

**Comment by fatboystring** [comment]
@dammitjim I've added some basic handler docs

**Pull request merged**